.github/workflows/tpu-testing.yml

```diff
@@ -0,0 +1,105 @@
+name: Run TPU tests on current PR.
+
+on:
+  pull_request:
+    branches:
+    - master
+
+env:
+  PROJECT_ID: ${{ secrets.GKE_PROJECT }}
+  GKE_CLUSTER: lightning-cluster   # TODO: update to cluster name.
+  GKE_ZONE: us-central1-a  # TODO: update to cluster zone.
```

Comment:
is the todo here still valid?

Reply:
no, this is done. These vars are working